src/sql-server-util/src/cdc.rs

```diff
@@ -209,7 +235,10 @@ impl<'a> CdcStream<'a> {
                     }
 
                     // Increment our LSN (`get_changes` is inclusive).
-                    let next_lsn = crate::inspect::increment_lsn(self.client, new_lsn).await?;
+                    max_observed_lsn = db_max_lsn;
+                    // TODO(sql_server2): We should occassionally check to see how close the LSN we
+                    // generate is to the LSN returned from incrementing via SQL Server itself.
```

Comment:
Out of curiosity, what will we do with this information?

Reply:
Not much, but it might be interesting to observe how the LSN of the upstream advances just for our own understanding